src/components/Menu/ActionSingle.vue

```diff
@@ -98,20 +98,11 @@ export default {
 			icon,
 			isItem,
 			runAction,
-			tooltip,
 			label,
 		} = this
 
 		const { class: classes, ...attrs } = bindState
 
-		// do not use tooltip if is a item of action list
-		const directives = isItem
-			? []
-			: [{
-				name: 'tooltip',
-				value: tooltip,
-			}]
```

Comment:
Currently `title` (that replaces the tooltip) has `isItem ? undefined : label`, not `tooltip` as a value.

https://github.com/nextcloud/text/blob/4a707d05643c182e114b9c1a44d718b9c26bc877/src/components/Menu/BaseActionEntry.js#L68-L73

Should here `title` also be `tooltip` and not `label`?

Reply:
Hmm, I think, yes, it should. It shows the shortcut on desktop

Reply:
PR Updated